src/JustSaying/Messaging/MessageProcessingStrategies/MessageConstants.cs

```diff
@@ -3,6 +3,6 @@ namespace JustSaying.Messaging.MessageProcessingStrategies
     public static class MessageConstants
```

Comment:
Maybe we should rename this `MessageDefaults` as none of them are actually constant?

Reply:
I think MessageConstants still makes sense here, even though it's not a `const`

Reply:
Done - this is public though, so will probably break people.